src/chains/optimism/nodes.ts

```diff
@@ -0,0 +1,6 @@
+import { consensusNodes as mainnetConsensusNodes } from '@/chains/mainnet/nodes/consensus';
+import { executionNodes as mainnetExecutionNodes } from '@/chains/mainnet/nodes/execution';
+import { Node } from '@/types';
+
+export const executionNodes: Node[] = mainnetExecutionNodes;
+export const consensusNodes: Node[] = mainnetConsensusNodes;
```

Comment:
L2s don't have consensus in the way L1 does, so this can be an empty list. For optimism execution there is:
- https://github.com/ethereum-optimism/op-geth
- https://github.com/testinprod-io/op-erigon
- https://github.com/paradigmxyz/reth (it has an optimism mode, see https://github.com/paradigmxyz/reth/blob/08dff3642da3ea0445e5a1911cff8c0cdecc9fc8/book/run/optimism.md)
- https://github.com/NethermindEth/nethermind (also support optimism)

Only the last 2 overlap with mainnet, so we'll need to update the execution array to just have those 4

Reply:
Yeah you're right, I just added those as placeholders for the moment. Will modify :)

Reply:
> Yeah you're right, I just added those as placeholders for the moment. Will modify :)

Ah sorry, the PR was not in draft so didn't realize they were placeholders. Going forward let's keep PRs in draft until all placeholders, etc. are removed to make it more clear (and/or indicate with comments)

And yea op-node is _analogous_ to the L1 consensus clients, but there is no consensus so it's not actually a consensus client. So for now I'd say let's remove op-node for simplicity/clarity.

Reply:
Removed in 2e6c1a1

Reply:
Yes my bad the PR should have been in draft :)